src/components/ActionBar/src/ActionBarButton.vue

```diff
@@ -155,15 +163,27 @@ export default {
 			if (this.$scopedSlots.information) {
 				return false;
 			}
-			const zero = 0;
 			const children = (this.$slots.default || []).filter(
-				(vnode) => vnode.tag || vnode.text.trim().length > zero,
+				(vnode) => vnode.tag || vnode.text.trim().length > 0,
 			);
 			const singleChild = 1;
 			const firstChildIndex = 0;
 			return children.length === singleChild && children[firstChildIndex].tag;
 		},
 
+		hasMainAndLabelText() {
+			if (!this.$scopedSlots.information) {
+				return false;
+			}
+			const main = (this.$slots.default || []).filter(
+				(vnode) => vnode.tag || vnode.text.trim().length > 0,
+			);
+			const info = (this.$scopedSlots.information() || []).filter(
```

Comment:
since we're using this pattern in 3 different places within this component can we refactor it out into its own function? e.g.
```
function getNumberOfContentfulVnodes(vnodes) {
    return (vnodes || []).filter( /* etc */);
}
```